core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/progress/listener/LoggingProgressListener.java

```diff
@@ -64,6 +64,7 @@ public void requestPrepared(Context.RequestPrepared context) {
 
     @Override
     public void requestHeaderSent(Context.RequestHeaderSent context) {
+        log.info(() -> "Started Upload...");
```

Comment:
Do we want to keep this log?

Reply:
removed it, we wouldnt be needing any logs out there.